translate.go

```diff
@@ -90,6 +89,12 @@ func OptTranslateFileMapSize(mapSize int) TranslateFileOption {
 		return nil
 	}
 }
+func OptTranslateFileLogger(l Logger) TranslateFileOption {
```

Comment:
This doesn't seem to actually be used?

Reply:
right, but presumably we would want to pass the logger in that's used by the rest of the Pilosa components.